devicetypes/krlaframboise/aeotec-doorbell-6.src/aeotec-doorbell-6.groovy

```diff
@@ -0,0 +1,1836 @@
+/**
+ *  Aeotec Doorbell 6 v1.1.1
+ *  (Model: ZW162-A)
+ *
+ *  Author: 
+ *    Kevin LaFramboise (krlaframboise)
+ *
+ *	URL to Documentation: https://community.smartthings.com/t/release-aeotec-doorbell-6/165030
+ *    
+ *
+ *  Changelog:
+ *
+ *    1.1.1 (06/05/2019)
+ *      - Fixed issue with default settings not populating after first save.
+ *
+ *    1.1 (06/01/2019)
+ *      - Initial Release
+ *
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+*/
+import groovy.transform.Field
+
+@Field LinkedHashMap VOLUME = [mute:0]
+@Field LinkedHashMap TONE = [longest:22, shortest:29, tamper:17]
+@Field LinkedHashMap INTERVAL_BETWEEN = [notStopping:0]
+@Field LinkedHashMap LIGHT_EFFECT = [off:1, on:2, flash:32, strobe:64]
+@Field LinkedHashMap CONTINUOUS_PLAY_COUNT = [continuous:0]
+@Field LinkedHashMap INTERCEPT_LENGTH = [actualToneLength:0]
+@Field LinkedHashMap PLAY_ASSOC = [groupId:7, endpoint:6]
+@Field LinkedHashMap TAMPER_ASSOC = [groupId:3, endpoint:2]
+@Field LinkedHashMap BUTTON_MODE = [normal:0, pairing:1, unpairing:2]
+@Field LinkedHashMap BUTTON1 = [num:1, infoParamNum:52, groupParamNum:3, endpoint:3, assocGroupId:4, pairingMode:1]
+@Field LinkedHashMap BUTTON2 = [num:2, infoParamNum:53, groupParamNum:4, endpoint:4, assocGroupId:5, pairingMode:2]
+@Field LinkedHashMap BUTTON3 = [num:3, infoParamNum:54, groupParamNum:5, endpoint:5, assocGroupId:6, pairingMode:4]
+ 
+metadata {
+	definition (
+		name: "Aeotec Doorbell 6", 
+		namespace: "krlaframboise", 
+		author: "Kevin LaFramboise",
+		ocfDeviceType: "x.com.st.d.siren",
+		vid:"generic-siren"
+	) {
+		capability "Actuator"
+		capability "Alarm"
+		capability "Switch"		
+		capability "Audio Notification"
+		capability "Music Player"
+		capability "Speech Synthesis"
+		capability "Switch Level"		
+		capability "Tamper Alert"
+		capability "Tone"
+		// capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
+				
+		attribute "lastCheckIn", "string"
```

Comment:
@ccheng-aeotec the attributes currently don't work with our new SmartThings app. in the SmartThings classic app the attributes are used. 
can you please let us know if the device was tested with the new app